lib/dom/genDomScripts.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * We have a collection of typescript functions that we need to run in the browser.
+ * First, we build them into a single js file
+ * Second, due to framework differences we need to get our script content as a string to avoid pathing issues due to file routing in frameworks like Next.js
+ * Playwright allows us to pass in script content directly as a string instead of reading a file from a path
+ * https://github.com/browserbase/stagehand/issues/180
+ *
+ * We can't rely on the normal build process for stagehand, because we need our script content as a string so that the import *just works*
+ */
+const fs = require("fs");
```

Comment:
Do we want to be using `require` over `import`?

Reply:
yeah that was claude, will clean up